cli/popper/commands/cmd_stages.py

```diff
@@ -0,0 +1,45 @@
+import click
+import os
+import popper.utils as pu
+
+from popper.cli import pass_context
+from os.path import isfile
+
+@click.command('stages', short_help='Comma-separated list of stages to a pipeline')
+@click.argument('pipeline', required=True)
+@click.option(
+    '--set',
+    help="Redefine the stages of a pipeline",
+    required=False
+)
+@pass_context
+def cli(ctx, pipeline, set):
+    """Manipulates the stages of a pipeline.The stages YAML entry specifies
+    the sequence of stages that are executed by the popper check command.When
+    a new pipeline is created, the popper init generates scaffold scripts for
+    setup.sh,run.sh,post-run.sh,validate.sh,teardown.sh by default.(See popper
+    init --help for more)
+
+    Example:
+
+    popper stages #Shows the stages of a pipeline
+
+    popper stages --set=one,two,three #Redefine the stages of a pipeline
+    """
+    config = pu.read_config()
+
+    if not set:
+        pu.info(str(config['pipelines'][pipeline]['stages']))
+
+    if set:
+        for s in stages.split(','):
+            s_filename = os.path.join(pipeline_path, s)
+            if not isfile(s_filename) and not isfile(s_filename+'.sh'):
```

Comment:
Use the get_filename() method when https://github.com/systemslab/popper/pull/246 is merged.

Reply:
Okay, i will make the changes when #246 will get merged.

Reply:
cool, let us know if you have any question